frontends/web/src/components/terms/btcdirect-terms.tsx

```diff
@@ -0,0 +1,107 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useTranslation } from 'react-i18next';
+import { ChangeEvent } from 'react';
+import { Button, Checkbox } from '@/components/forms';
+import { setConfig } from '@/utils/config';
+import style from './terms.module.css';
+import { i18n } from '@/i18n/i18n';
+import { A } from '../anchor/anchor';
+
+type TProps = {
+  onContinue: () => void;
+}
+
+export const BTCDirectTerms = ({ onContinue }: TProps) => {
+  const { t } = useTranslation();
+  const handleSkipDisclaimer = (e: ChangeEvent<HTMLInputElement>) => {
+    setConfig({ frontend: { skipBTCDirectDisclaimer: e.target.checked } });
+  };
+
+  const getPrivacyLink = () => {
+    switch (i18n.resolvedLanguage) {
+    case 'de':
+      return 'https://btcdirect.eu/de-at/datenschutzenklaerung';
+    case 'nl':
+      return 'https://btcdirect.eu/nl-nl/privacy-policy';
+    case 'es':
+      return 'https://btcdirect.eu/es-es/privacy-policy';
+    case 'fr':
+      return 'https://btcdirect.eu/fr-fr/privacy-policy';
+    default:
+      return 'https://btcdirect.eu/en-eu/privacy-policy';
+    }
+  };
+
+  return (
+    <div className={style.disclaimerContainer}>
+      <div className={style.disclaimer}>
+        <h2 className={style.title}>{t('buy.exchange.infoContent.btcdirect.disclaimer.partnership.title')}</h2>
+        <p>{t('buy.exchange.infoContent.btcdirect.disclaimer.partnership.text')}</p>
+        <h2 className={style.title}>{t('buy.exchange.infoContent.btcdirect.disclaimer.personal.title')}</h2>
+        <p>{t('buy.exchange.infoContent.btcdirect.disclaimer.personal.text')}</p>
+        <h2 className={style.title}>{t('buy.exchange.infoContent.btcdirect.disclaimer.paymentMethods.title')}</h2>
+        <ul>
+          <li>
+            <p>
+              <strong>{t('buy.exchange.infoContent.btcdirect.disclaimer.paymentMethods.buy')}</strong>
+              &nbsp;
+              {t('buy.exchange.infoContent.btcdirect.disclaimer.paymentMethods.buy2')}
+            </p>
+          </li>
+          <li>
+            <p>
+              <strong>{t('buy.exchange.infoContent.btcdirect.disclaimer.paymentMethods.sell')}</strong>
+              &nbsp;
+              {t('buy.exchange.infoContent.btcdirect.disclaimer.paymentMethods.sell2')}
+            </p>
+          </li>
+        </ul>
+        <p>{t('buy.exchange.infoContent.btcdirect.disclaimer.paymentMethods.fee')}</p>
+        <h2 className={style.title}>{t('buy.exchange.infoContent.btcdirect.disclaimer.security.title')}</h2>
+        <p>{t('buy.exchange.infoContent.btcdirect.disclaimer.security.text')}</p>
+        <p>
+          <A href="https://bitbox.swiss/bitbox02/threat-model/">
```

Comment:
note for myself: threat-model is english only so it doesn't need to point to different langs.